build/configs/rtl8720e/scripts/rlx8720e_img2.ld

```diff
@@ -86,12 +86,12 @@ MEMORY
 	KM4_HEAP_EXT (rwx) :                    ORIGIN = 0x2007A000, LENGTH = 0x200C0000 - 0x2007A000	/* KM4 HEAP EXT: 280k */
 
 	/* KM4 Non-secure PSRAM */
-	KM4_BD_PSRAM (rwx) :                    ORIGIN = 0x60000020, LENGTH = 0x60379000 - 0x60000020	/* KM4 MAIN RAM NS : 3556 */
+	KM4_BD_PSRAM (rwx) :                    ORIGIN = 0x60000020, LENGTH = 0x6037E000 - 0x60000020	/* KM4 MAIN RAM NS : 3575 */
```

Comment:
You reduce `KM4 BD RAM S` 20K, but this increases 19K. Is there any reason?

Reply:
Hi 

Currently, we are doing the optimization to reserve Heap Region to 4MB. 
We had done the optimization code for the TrustZone part, so It can reduce the KM4_BD_PSRAM_TZ_S 20k.
We use the saved 20k size from KM4_BD_PSRAM_TZ_S region and add into the KM4_BD_PSRAM region.

![image](https://user-images.githubusercontent.com/94035727/209113155-179893e5-259d-4fd5-acf1-77d2b88f2ef0.png)
Hope this layout can have a better understanding.

Thank You.

Reply:
Hello @ZhenBei-Sin . What i said is mismatch of the values, decrease 20k in tz area and increase 19k in heap.

Reply:
Hello @sunghan-chang ,
The comment for KM4_BD_PSRAM seems can be confused.

KM4_BD_PSRAM_TZ_S (rwx) :               ORIGIN = 0x70383020, LENGTH = 0x703CE000 - 0x70383020   /* KM4 BD RAM S: 300k */
For this , exact size is 299.9K but zhenbei commented to 300K
But,
KM4_BD_PSRAM (rwx) :                    ORIGIN = 0x60000020, LENGTH = 0x6037E000 - 0x60000020   /* KM4 MAIN RAM NS : 3575 */
**for this, exact size is 3575.9K but zhenbei commented to 3575K . not 3576K**

So, KM4 BD_PSRAM size was actually increased 20K.
We will replace this comment next time.